requirements.txt

```diff
@@ -26,12 +26,12 @@ django-debug-toolbar==1.6
 django-dotenv==1.4.1
 django-easy-pdf==0.1.1
 django-localflavor==1.4.1
-django-modelcluster==2.0
+django-modelcluster==3.0
```

Comment:
can we use the syntax that will upgrade to latest minor version or bug fix version or something like that? i don't see why we should pin it to the x.0 version. for wagtail we are pulling in the latest minor version for a specific major version.

see related discussion: https://github.com/Our-Revolution/site/pull/227/files#r189662101

Reply:
for example i see a 3.0.1 version with a bug fix available